tests/integration/workspace_access/test_generic.py

```diff
@@ -19,20 +20,28 @@
     models_root_page,
     tokens_and_passwords,
 )
-from databricks.labs.ucx.workspace_access.groups import MigratedGroup
+from databricks.labs.ucx.workspace_access.groups import MigrationState
+from databricks.labs.ucx.workspace_access.manager import PermissionManager
 
 from . import apply_tasks
 
 
+@pytest.mark.parametrize("is_experimental", [True, False])
 @retried(on=[NotFound], timeout=timedelta(minutes=3))
-def test_instance_pools(ws, make_group, make_instance_pool, make_instance_pool_permissions):
-    group_a = make_group()
-    group_b = make_group()
+def test_instance_pools(
+    ws: WorkspaceClient,
+    permission_manager: PermissionManager,
+    make_migrated_group,
+    make_instance_pool,
+    make_instance_pool_permissions,
+    is_experimental: bool,
+):
+    migrated_group = make_migrated_group()
```

Comment:
```suggestion
    migrated_group,
    make_instance_pool,
    make_instance_pool_permissions,
    is_experimental: bool,
):
```
nit: what if we switch from fixture factory (`make_*`) to just a fixture?